### ISSUE TITLE
[Hacktoberfest] Covert PayPalButtons.test.js to TypeScript

### ISSUE THREAD

**Issue by gregjopa** [paypal/react-paypal-js#167]
We have a bunch of tests that are still using plain JavaScript even though this library is written in TypeScript.

For this issue, convert the [PayPalButtons.test.js](https://github.com/paypal/react-paypal-js/blob/main/src/components/PayPalButtons.test.js) to TypeScript.

**Comment by Torfab** [comment]
Hi Greg,
I can do this 👍

**Comment by Abhijay007** [comment]
Hi, @gregjopa can you please suggest to me some issues like these (conversion issues )contribute !!

**Comment by shubhangi013** [comment]
Hey! @gregjopa Please assign me this issue.

**Comment by davidalpert** [comment]
you may want to wait until we merge #174 (or cherry-pick the `package.json` changes from that PR) as I ran into some missing types and a mocking library which will probably be helpful for all the test file conversions.

**Comment by Torfab** [comment]
yes, i started working on it and noticed some mismatching in types, thank you @davidalpert
